src/util/readerSettings.ts

```diff
@@ -10,14 +10,13 @@ import { getMetadataFrom, requestUpdateMangaMetadata, requestUpdateServerMetadat
 import { useQuery } from 'util/client';
 import { IManga, IMetadata, IMetadataHolder, IReaderSettings, MetadataKeyValuePair } from 'typings';
 
-export const getDefaultSettings = (forceUndefined: boolean = false) =>
-    ({
-        staticNav: forceUndefined ? undefined : false,
-        showPageNumber: forceUndefined ? undefined : true,
-        continuesPageGap: forceUndefined ? undefined : false,
-        loadNextOnEnding: forceUndefined ? undefined : false,
-        readerType: forceUndefined ? undefined : 'ContinuesVertical',
-    } as IReaderSettings);
+export const getDefaultSettings = (forceUndefined: boolean = false): Partial<IReaderSettings> => ({
```

Comment:
I actually is my code 🤣 (the `forceUndefined`)

It's being used for getting the reader settings from the metadata of the manga with undefined default values.
but ofc that only worked because of the incorrect typing and I didn't want to create the settings object again in a different place.

but probably should be done the correct way

Reply:
I will change it in another pr

Reply:
https://github.com/Suwayomi/Tachidesk-WebUI/pull/268